week-2/Alam - Assignment 2/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+	<head>
+			<title>Madiha Alam</title>
+	</head>
+	<body>
+		<h1>Design Portfolio</h1>
+		<h3>Madiha Alam</h3>
+		<p>I would like to create this website to display my past work in design as well as have a space to upload any future projects. </br>As the semesters go on, my work accumulates and I would like one place to display it to anyone who would like to see it. </br>This will come in particularly useful when I am applying for jobs and internships.</p>
+
+	<hr/>
+
+		<h2>Content</h2>
+		<p>These are the different categories I would like to focus on in this website:</p>
+
+		<ol>
+			<li><b>Digital Art</b>

```

Comment:
In this case, you'll want to close the `<li>` before you start the `<ul>`, since the `<ul>` is within the `<ol>`, it will still appear as a sub list